OmgvaPOS/src/HelperUtils/AuthorizationHandler.cs

```diff
@@ -0,0 +1,67 @@
+﻿using OmgvaPOS.UserManagement.Enums;
+
+namespace OmgvaPOS.HelperUtils
+{
+    public static class AuthorizationHandler
+    {
+        public static bool CanManageBusiness(string? tokenString, long businessId)
+        {
+            var token = JwtTokenHandler.GetTokenDetails(tokenString);
+            
+            if (token.BusinessId != businessId)
+                return false;
+
+            if (token.UserRole == UserRole.Admin 
+                || token.UserRole == UserRole.Owner 
+                || token.UserRole == UserRole.Employee)
+            {
+                return true;
+            }
+            
+            return false;
+        }
+
+        public static bool CanManageUser(string? tokenString, long businessId, long userId)
+        {
+            var token = JwtTokenHandler.GetTokenDetails(tokenString);
+
+            if (token.UserRole == UserRole.Admin)
+                return true;
+
+            if (token.UserRole == UserRole.Employee && token.UserId == userId)
+                return true;
+
+            if (token.UserRole == UserRole.Owner && token.BusinessId == businessId)
+                return true;
+            
+            return false;
+        }
+
+        public static bool CanDeleteUser(string? tokenString, long businessId, long userId)
+        {
```

Comment:
This method was meant to check if id matches the user and forbid the user to delete his account. Right now it no longer accomplishes this task at all. Revert this (though admin probably could delete itself previously, if yes, it should be fixed)

Reply:
Why would we not let a user to delete their own account?

Reply:
Because they cannot. IRL, if you are leaving a company, they will delete everything for you.

Reply:
Also, this would be very weird feature. What happens when you delete yourelf? Immediately get kicked from the system?

Reply:
Well.. sort of. Every platform where I tried to delete myself it's possible. Granted, they make sure you REALLY want to delete yourself by confirming a few times or offering some other solution like making your account inactive for some time

Reply:
However in our case - yeah, this does not really make sense. When you leave a company they delete everything for you

Reply:
Updated. We decided no one can delete their own account. Employee cannot delete _any_ account. Owner can delete account from their own business, admin can delete any account (except their own, offcourse)